index.bs

```diff
@@ -346,20 +346,99 @@ interface mixin DatagramTransport {
    but large enough to avoid dropping packets when for the stream is not read
    for short periods of time (due to the reader being paused).
 
-# `WebTransport` Mixin #  {#web-transport}
+# `WebTransport` Interface #  {#web-transport}
 
-The `WebTransport` includes the methods common to all transports, such as
-state, state changes, and the ability to close the transport.
+The `WebTransport` provides a unified interface to all client-server transports
+that are supported by the WebTransport API.  It implements all of the transport
+mixins ({{UnidirectionalStreamsTransport}}, {{BidirectionalStreamsTransport}},
+{{DatagramTransport}}), as well as stats and transport state information.
```

Comment:
Do we still need these mixins given that we only have WebTransport now?

Reply:
We still need some way to leave a plug for a peer-to-peer version of WebTransport, but I suspect we don't need all three of them.  Let's fix this in a follow-up, though.